src/CircleMenu/CircleMenuItem/CircleMenuItem.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * The CircleMenuItem.
+ *
+ * @class CircleMenuItem
+ * @extends React.Component
+ */
+export class CircleMenuItem extends React.Component {
+
+  _ref = null;
+
+  static propTypes = {
+    /**
+     * The duration of the animation in milliseconds. Pass 0 to avoid animation.
+     * Default is 300.
+     *
+     * @type {Number}
+     */
+    animationDuration: PropTypes.number,
+    /**
+     * The index of the current CircleMenuItem.
+     */
+    idx: PropTypes.number,
+    /**
+     * The radius of the CircleMenu.
+     *
+     * @type {Number}
+     */
+    radius: PropTypes.number,
+    /**
+     * The children of the CircleMenuItem. Should be just one Node.
+     *
+     * @type {Node}
+     */
+    children: PropTypes.node,
+    /**
+     * The rotation Angel
+     *
+     * @type {Number}
+     */
+    rotationAngle: PropTypes.number
+  };
+
+  static defaultProps = {
+    animationDuration: 300,
+    visible: false
+  };
+
+  /**
+   * Create the CircleMenuItem.
+   *
+   * @constructs CircleMenuItem
+   */
+  constructor () {
+    super();
```

Comment:
Pass props.

Reply:
You don't need to if you don't wan't to access `this.props` in the constructor... I removed the constructor anyways as it does nothing. :smile: